Keep compatibility for old getcap output formats

fkasler opened the pull request:
##### SUMMARY
getcap on some systems produces output like:
/usr/bin/node cap_net_bind_service=eip

Instead of:
/usr/bin/node = cap_net_bind_service=eip

This fix adds an elif to parse this type of output as a fallback

##### ISSUE TYPE
<!--- Pick one below and delete the rest -->
- Feature Pull Request


##### COMPONENT NAME
parse old getcap output formats

##### ADDITIONAL INFORMATION
Running:
```getcap /usr/bin/node```

on a Kali box with eip already set, produces:
/usr/bin/node cap_net_bind_service=eip

and triggers the following ansible error:
"Unable to get capabilities of /usr/bin/node"

This change fixes the parser to account for this output format

felixfontein commented:
What's the difference to `caps = stdout.split()[1].strip().split()` in the `else` branch? That will also handle a single space instead of `' = '`.

In any case, what exactly does `getcap` return if there are capabilities with different op/flags? TBH the code in `else` (and in your patch) looks a bit strange to me, since the `split()` at the end will proably never return a list with more than one element. @mator since you authored #1061, can you shed light on this?

fkasler commented:
> What's the difference to `caps = stdout.split()[1].strip().split()` in the `else` branch? That will also handle a single space instead of `' = '`.
> 
> In any case, what exactly does `getcap` return if there are capabilities with different op/flags? TBH the code in `else` (and in your patch) looks a bit strange to me, since the `split()` at the end will proably never return a list with more than one element. @mator since you authored #1061, can you shed light on this?

I actually transposed this fix from the capabilities.py that was install by pip on my system (Mac) and appears to be the version modified in #1061. I did not realize his fix "should" work too. You can reject my fix, but I am now wondering why the latest version of the file is not installed by pip. Any advice welcome.

felixfontein commented:
About pip: I guess that depends what you install with pip, and which version is installed. This collection cannot be directly installed by pip, so I guess you mean the `ansible` PyPi package? Which version of it do you have installed? And did you manually install community.general as well? (If you install a collection manually, it will hide the version in the `ansible` package, so even if you update the `ansible` package, you keep using the version of that collection you manually installed.)

fkasler commented:
Thanks for the review Felix (and sorry for the hassle). This error has been a recurring theme for my team, so I will look into our Ansible install instructions and see if I can remediate by specifying the right collection version.

felixfontein commented:
@fkasler no problem, it's better to look at such issues too often than too little ;-)
